libs/base/System/FFI.idr

```diff
@@ -52,6 +52,7 @@ setField s n val = primIO (prim__setField s n fieldok val)
 prim__malloc : (size : Int) -> PrimIO AnyPtr
 
 %foreign "C:idris2_free, libidris2_support, idris_memory.h"
+         "javascript:lambda:()=>undefined"
```

Comment:
won't this turn use of `prim_free` from a compile-time error to a runtime error? Is this what we want?

Reply:
My understanding is that there isn’t a C FFI in node, so all of these pointers are memory managed. (E.g. readLine is getting a proper GC managed JavaScript string instead of a void*.)

I don’t believe there are any naked pointers with the JavaScript backend, but if there were, the failure mode here would be to leak memory (this is a noop free).

Reply:
Ah yes that makes sense!